### PR TITLE
Update package-lock.json

### LINKED ISSUE #196: Use sass (dart-sass) instead of node-sass
**Description**:
Use [`sass(dart-sass)`](https://www.npmjs.com/package/sass) instead of `node-sass`.

**Why**:
- `node-sass` is dependent on `libsass`, which is written in c/c++, so we have to compile using `node-gyp`, which creates a dependence on the node version.
- `libsass` and `node-sass` is deprecated. See https://sass-lang.com/blog/libsass-is-deprecated
- The Sass team recommends using the `dart-sass`, and as far as I know, it is drop-in replacement for `node-sass`.

### PR DESCRIPTION
**Pull request by metleeha**
Delete deprecated lib "node-sass" & Add latest "sass"

#### What this PR does / why we need it 
Since `lib-sass` and `node-sass` have been deprecated, they fall back to the `dart-sass`.

#### Any background context you want to provide?
I installed 1.53.0, the most recent version that can be found in the dart-sass github repo.
Since I could not find any code used by importing the `node-sass` library, I didn't modify any other code. 

#### What are the relevant tickets?

Fixes #196

### Checklist
- [x] Added relevant tests or not required
- [ ] Didn't break anything

### CODE REVIEW

**Review by ppeeou — APPROVED** [review]
lgtm

**Review by blurfx — APPROVED** [review]
LGTM 👍

**Comment by blurfx** [comment]
Hmm. It might be helpful to use `npm ci` command instead of `npm install` on github action workflow.

**Comment by blurfx** [comment]
Would you like to delete node_modules and package-lock.json, install npm locally, and commit? [[ref]](https://github.com/fsevents/fsevents/issues/278#issuecomment-538449044)

**Comment by metleeha** [comment]
>  I deleted the package-lock.json and node-modules file, reinstalled it with the npm i -D fsevents command, and committed it.

**Comment by blurfx** [comment]
Great 👍 . Could you resolve the conflict? I think you can just use the new `package-lock.json`.

File: package.json [hunk 1]
```diff
@@ -99,8 +99,5 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
-  },
-  "devDependencies": {
-    "fsevents": "^2.3.2"
```
Comment (blurfx) [inline package.json]: dev dependencies was remove while resolving the conflict. 😮

**Comment by blurfx** [comment]
I think we should use `-f` flag for now. (We added the `-f` in #202. )

Or, could you please investigate cross-platform build is possible in any other way? (without adding `-f`?)

**Pull request merged**